docs/references/react/use-user.mdx

```diff
@@ -83,28 +79,26 @@ export default function Home() {
 
   return (
     <>
-      <button onClick={updateUser}>Click me to update your name</button>
+      <button onClick={updateUser}>Update your name</button>
       <p>user.firstName: {user?.firstName}</p>
       <p>user.lastName: {user?.lastName}</p>
     </>
   )
 }
 ```
 
-### Reload user data with the `useUser()` hook
-
-To retrieve the latest user data after updating the user elsewhere, use the `user.reload()` method.
+### Reload user data
 
-For more information on the `reload()` method, see the [`User`](/docs/references/javascript/user/user#reload) reference.
+Use [`reload()`](/docs/references/javascript/user/user#reload) to get the latest user's information via an external API endpoint.
```

Comment:
it should provide more context.
the useUser() hook gives access to the user object, which is how you access the reload method.
useUser() does not give you direct access to the reload method